proposals/lip-0049.md

```diff
@@ -273,6 +273,9 @@ def apply(trs: Transaction, ccm: CCM) -> None:
     else:
         crossChainCommand = cross-chain command associated with (ccm.module, ccm.crossChainCommand)
         try:  
+            #Decode and validate the parameterSchema associated with ccm module or crossChainCommand
```

Comment:
```suggestion
            #Decode and validate the parameter schema associated with ccm module and crossChainCommand
```

Reply:
Also, if you define the params schema as I suggest, this comment is probably superfluous